File: src/Mova21AppBackend.Data/Models/WeatherEntry.cs
using System.ComponentModel.DataAnnotations;

namespace Mova21AppBackend.Data.Models
{
    public class WeatherEntry
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public DayTime DayTime { get; set; }
        public double Temperature { get; set; }
        public WeatherType Weather { get; set; }
    }

    public enum WeatherType
    {
        Cloud,
        CloudSun,
        CloudSunRain,
        CloudRain,
        Sun,
        Thunderstorm,
        Fog,
        Snow
    }

    public enum DayTime
    {
        Morning,
        Midday,
        Evening,
        Night
    }
}

File: src/Mova21AppBackend/ClientApp/src/app/shared/models/weatherEntry.ts
import { WeatherType } from "./weatherType";
import { DayTime } from "./dayTime";

export class WeatherEntry {
  id = 0;
  date: Date = new Date();
  dayTime: DayTime = DayTime.Morning;
  temperature: number = 0;
  weather: WeatherType = WeatherType.Sun;
}
